src/utils/typeChecking.test.js

```diff
@@ -1,6 +1,52 @@
 import { expect } from 'chai';
 
-import { parseIntExact } from './typeChecking';
+import {
+  isString,
+  parseIntExact
```

Comment:
I'm confused... `parseIntExact` doesn't sound like a function whose purpose is to check a type... :P